proto/trailers.proto

```diff
@@ -0,0 +1,34 @@
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
+option java_package = "momento.trailers";
+option csharp_namespace = "Momento.Protos.Trailers";
+
+package trailers;
+
+// Found on the `retry_disposition` trailer for error responses, when the value is known by the service
+enum RetryDisposition {
+     // This rpc is safe to retry, even if it is non-idempotent. It was not executed by the server.
+    retryable = 0;
+    // This rpc may be safe to retry, but it may have been applied.
+    // Non-idempotent commands should not be retried, unless configured explicitly.
+    // Idempotent commands should be considered eligible for retry.
+    possibly_retryable = 1;
```

Comment:
is there any case where we would want to have a `not_retryable` here, or where the server would know that at the time it created the response?

Reply:
okay. just keep in mind that once the sdks are updated to use these, then any changes you make on the server will require additional sdk updates.